packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -186,9 +243,17 @@ export async function generateImageData({
   }
 
   if (placeholder === `blurred`) {
+    const placeholderWidth =
+      args.blurredOptions?.width || DEFAULT_BLURRED_IMAGE_WIDTH
```

Comment:
Shouldn't this default to unscaledWidth?

Reply:
No, it's the tiny low res preview, so usually 20px wide